### PR TITLE
Test

### PR DESCRIPTION
**Pull request by suadsuljovic**
sdfsdfds

### CODE REVIEW

**Comment by suadsuljovic** [comment]
sdfsdf

File: README.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ How to use this guide can be found on [Pick your poison](project-building-steps/
 All the projects can be found in the [projects](projects) folder.
 
 You can contribute or fork the repository anytime.
+
```
Comment (suadsuljovic) [inline README.md:16]: This is not good

File: README.md [hunk 2]
```diff
@@ -13,3 +13,5 @@ How to use this guide can be found on [Pick your poison](project-building-steps/
...
+test
```
Comment (suadsuljovic) [inline README.md:17]: second comment

**Review by hamzaplojovic — APPROVED** [review]

**Review by hamzaplojovic — APPROVED** [review]

**Review by hamzaplojovic — APPROVED** [review]
.

**Pull request closed without merging**